src/Traversal/Sdk/Sdk.targets

```diff
@@ -111,47 +111,52 @@
     <MSBuild Projects="@(ProjectReference)"
              BuildInParallel="$(BuildInParallel)"
              SkipNonexistentProjects="$(SkipNonexistentProjects)"
-             SkipNonexistentTargets="$(SkipNonexistentTargets)" />
+             SkipNonexistentTargets="$(SkipNonexistentTargets)"
+             ContinueOnError="$(ContinueOnError)" />
   </Target>
 
   <Target Name="Clean"
           DependsOnTargets="$(CleanDependsOn)"
           Condition=" '$(IsGraphBuild)' != 'true' ">
     <MSBuild Projects="@(ProjectReference)"
              Targets="Clean"
-             BuildInParallel="$(BuildInParallel)"
+             BuildInParallel="$([MSBuild]::ValueOrDefault('$(CleanInParallel)', '$(BuildInParallel)')"
              SkipNonexistentProjects="$(SkipNonexistentProjects)"
-             SkipNonexistentTargets="$(SkipNonexistentTargets)" />
+             SkipNonexistentTargets="$(SkipNonexistentTargets)"
+             ContinueOnError="$([MSBuild]::ValueOrDefault('$(CleanContinueOnError)', '$(ContinueOnError)')" />
   </Target>
 
   <Target Name="Test"
           DependsOnTargets="$(TestDependsOn)"
           Condition=" '$(IsGraphBuild)' != 'true' ">
     <MSBuild Projects="@(ProjectReference)"
              Targets="Test"
-             BuildInParallel="$(BuildInParallel)"
+             BuildInParallel="$([MSBuild]::ValueOrDefault('$(TestInParallel)', '$(BuildInParallel)')"
```

Comment:
You're missing a closing parenthesis

```diff
-BuildInParallel="$([MSBuild]::ValueOrDefault('$(TestInParallel)', '$(BuildInParallel)')"
+BuildInParallel="$([MSBuild]::ValueOrDefault('$(TestInParallel)', '$(BuildInParallel)'))"
```

And it turns out that this gets parsed down to an empty string and MSBuild still doesn't like that.

I think we'll have to default `BuildInParallel` and `ContinueOnError`:

```xml
<PropertyGroup>
  <BuildInParallel Condition="'$(BuildInParallel)' == ''">true</BuildInParallel>
  <ContinueOnErrorCondition="'$(ContinueOnError)' == ''">false</ContinueOnError>
</PropertyGroup>
```

Or we'll have to use a more complex syntax:
```xml
ContinueOnError="$([MSBuild]::ValueOrDefault($([MSBuild]::ValueOrDefault('$(CleanContinueOnError)', '$(ContinueOnError)')), 'false'))"
```

What do you think is better?

Reply:
Oh sorry, lol. 

> What do you think is better?

I believe we shouldn't set defaults here in the SDK... just my 5c. What if I revert this to the state before which requires the hardcoded props but still better than defaulting something here?

Reply:
These are already set in official SDKs.

https://github.com/microsoft/msbuild/blob/64c8e4a18e7cf7f064fbad304ea7ed877cdaa0a1/src/Tasks/Microsoft.Common.CurrentVersion.targets#L342
https://github.com/microsoft/msbuild/blob/64c8e4a18e7cf7f064fbad304ea7ed877cdaa0a1/src/Tasks/Microsoft.Common.CurrentVersion.targets#L610

So I'm comfortable setting them in Traversal as well.